andorCtrl_science.py

```diff
@@ -498,3 +498,25 @@ def acq_cube(self, N_frames, exptime, filename=None):
         self.pub.pprint("Image saved in '" + SAVEFILEPATH + final_filename + ".fits'\n")
 
         os.system("ds9 " + SAVEFILEPATH + final_filename + ".fits &")
+
+    def acq_cube_test(self, N_frames, exptime, filename=None):
+        self.set_exptime(exptime)
```

Comment:
@KBarjot So I think that it is this line that makes things not work. When we do a set_exp, the live run stops. and then restart after acquisition of the cube. (I think). So I'm going to try the function without it.. And we'll have to change the exptime before cube acquisition (which is not a bad thing)

Reply:
So I think the reason why it doesn't work is that there is a weird thing going on since the images are written in the "run" function. 
The multi_recv_data works well outside of the function. So I think what needs to be done is to call it in a separate python function

Reply:
Separate from what? The function run? The class andorCtrl? The script andorView?

Reply:
separate from the Andor control soft

Reply:
Ok i see.

Reply:
All right, I think it's good now.. 
You can test --> It's faster (x4 at least), and the other thing I wanna check is the type of data saved... Somehow the files saved with pymilk are 2 times less big than the other method

Reply:
I did commit all the changes.. I need to go now but if you're okay, and with another small review that I'll do later, we should be good to push this branch into the master

Reply:
I also want to add a function that takes several cubes at a time

Reply:
But I'll do that later, gotta go now :)

Reply:
Are the data lighter because you set np.float32 at initialisation? (May be it's float64 by default?)

Reply:
Yes that is correct.
@KBarjot  I have modified a bunch of things in the soft, I'm still debugging things so when it will be debug (most likely before the end of this week) I'll do a request of review from you to push it into the master branch)

Reply:
Ok, did you see my comments on your commits?

Reply:
I didn't, sorry.. I'm answering them :)